src/styles/mixin.scss

```diff
@@ -0,0 +1,27 @@
+@mixin catalog-padding {
+  padding-inline: 40px;
+
+  @media (min-width: 1200px) {
+    padding-inline: 128px;
+  }
+}
+
+@mixin page-grid {
+  grid-template-columns: 1fr;
+  gap: 46px 0;
+
+  @media (min-width: 480px) and (max-width: 767px) {
+    grid-template-columns: 1fr 1fr;
+    gap: 46px 48px;
+  }
+
+  @media (min-width: 768px) and (max-width: 1023) {
```

Comment:
There is a missing unit in the media query condition `(max-width: 1023)`. It should be `(max-width: 1023px)` to ensure the media query functions correctly.